app/api/recommend.py

```
@@ -110,30 +110,51 @@ def startup_event():
     cache_key = f"recommendation:user:{user_id}"
     cache_key_masked = mask_key(cache_key)
 
+    # 0. Redis 캐시 확인
     try:
         cached = r.get(cache_key)
         if cached:
             ttl = r.ttl(cache_key)
-            latency_ms = int((time.perf_counter() - t0) *1000)
-            logger.info(f"[CACHE][HIT] ttl={ttl} latencyMs={latency_ms} key={cache_key_masked}")
+            latency_ms = int((time.perf_counter() - start_time) *1000)
+            logger.info(f"[CACHE][HIT] ttl=$s key=%s", ttl, cache_key_masked, extra={
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)